### PR TITLE
updated install instructions for windows

### PR DESCRIPTION
**Pull request by gtfisher**
Signed-off-by: gtfisher <grant@chaosiq.io>

### CODE REVIEW

File: docs/getting-started/install/index.md [hunk 1]
```diff
@@ -36,6 +36,14 @@ chmod +x /usr/local/bin/reliably
 ```
 <CopyToClipboard />
 
+On Windows
+
+```console
+curl -L https://github.com/reliablyhq/cli/releases/latest/download/reliably-windows-amd64 -o %HOMEDRIVE%%HOMEPATH%/bin/reliably.exe
+```
+
```
Comment (mperrien) [inline docs/getting-started/install/index.md:44]: I haven't tested it with an empty line between the `<code>` block and the copy-to-clipboard component, but I assume it works!
(For example, at first I had the component _before_ the `<code>` block and it _had_ to have an empty line.

File: docs/getting-started/install/index.md [hunk 2]
```diff
@@ -165,7 +173,31 @@ You should see the current ```reliably``` cli version displayed.
 
 ### Windows 10
 
-* Download the latest [Windows 10 Release](https://github.com/reliablyhq/cli/releases/latest/download/reliably-windows-amd64) to a working directory.
+
+#### Install with curl
+
+The quick and easy way to install the  `reliably cli` on Windows is to use `curl`:
```
Comment (mperrien) [inline docs/getting-started/install/index.md:179]:
```suggestion
The quick and easy way to install the Reliably CLI on Windows is to use `curl`:
```
I _think_ we write it as normal text, as it's a product name, not as inline code, which we try to keep for actual CLI commands or other code tokens.

File: docs/getting-started/install/index.md [hunk 3]
```diff
@@ -165,7 +173,31 @@ You should see the current ```reliably``` cli version displayed.
...
+curl -L https://github.com/reliablyhq/cli/releases/latest/download/reliably-windows-amd64 -o %HOMEDRIVE%%HOMEPATH%/bin/reliably.exe
+```
+
+<CopyToClipboard />
+
+
+:::note Note
+The above command assumes you have the `curl` command available and you have a `bin` folder in your home directory. For ease of use, ensure the `bin` folder is on your path.
+
+You can install the `reliably cli` in any directory you like. Just update the `curl` command above as required.
```
Comment (mperrien) [inline docs/getting-started/install/index.md:191]:
```suggestion
You can install the Reliably CLI in any directory you like. Just update the `curl` command above as required.
```
Same as before.

File: docs/getting-started/install/index.md [hunk 4]
```diff
@@ -165,7 +173,31 @@ You should see the current ```reliably``` cli version displayed.
...
+
+`curl` is available as a Windows 10 command after version 1706. On other versions of Windows, you can [download curl] from the official curl website.
+:::
+
+[download curl]:https://curl.se/download.html
+
+#### Manual Install
+
+Download the latest [Windows 10 Release](https://github.com/reliablyhq/cli/releases/latest/download/reliably-windows-amd64) to a working directory.
```
Comment (mperrien) [inline docs/getting-started/install/index.md:200]: I found the beginning of the sentence hard to understand. I thought it was a link to download Windows.

Maybe "Download the latest Reliably CLI release for Windows 10" or "the latest release of the Reliably CLI for Windows 10"?

**Review by mperrien — CHANGES_REQUESTED** [review]
Minor comments.

File: docs/getting-started/install/index.md (hunk 1 above)
Comment (gtfisher) [reply docs/getting-started/install/index.md:44]: It works the same with the blank line in there, but I  will remove the blank line for consistency

**Pull request merged**